docs/Development/ReleaseNotes.md

```diff
@@ -236,6 +256,16 @@ This is a major update from VIC 4. The VIC 5.0.0 release aims to have nearly ide
 
 	Fixed a bug where volumetric heat capacity of water should be used in `func_canopy_energy_bal` (previously specific heat capacity was used).
 
+6. Fixes for the CESM driver ([GH#642](https://github.com/UW-Hydro/VIC/pull/642))
+
+	1. Using correct fill value datatypes in MPI Gather steps 
+	2. Updated state file name time step to be period-ending rather than period-beginning
+	3. Set the state file name to the RASM case ID
+	4. Removed decimal point for missing values for unsigned integers
+	5. Create dummy forcings when initializing the model (so that there is forcing data for the first time step)
+	6. Changed pressure units from kPa to Pa
+	7. Fixed bug that prevented using the correct local domain grid cells in `cesm_put_data.c`
+	8. Changed reference temperature units from Celsius to Kelvin in `cesm_put_data.c`
```

Comment:
These were changes in develop. The only CESM driver changes that should be in the 5.0.1 release are those for writing 2-d lat/lons to a state file, since that's relevant for the image driver.